recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -26,6 +26,8 @@ SRC_URI:append:pinephone = " \
     file://0001-pinephone-Add-volume_key-environment-variable.patch \
 "
 UBOOT_MAKE_TARGET:pinephone = "pinephone_defconfig all"
+UBOOT_MAKE_TARGET:pinetab2 = "pinetab2-rk3566_defconfig all"
```

Comment:
I just realized this line isn't needed, as we already use UBOOT_MACHINE variable in the machine conf file.

Reply:
@tofee will you remove it here then?

Reply:
@Tofee LGTM for the rest, all works

Reply:
Yes I'll amend this commit

Reply:
Ok, done